octoprint_mrbeam/iobeam/dust_manager.py

```diff
@@ -247,19 +270,13 @@ def _finish_test_fan_rpm(self):
         try:
             # Write to analytics if the values are valid
             if self._validate_values():
-                if len(self._last_rpm_values):
-                    rpm_average = sum(self._last_rpm_values) / len(
-                        self._last_rpm_values
-                    )
-                else:
-                    rpm_average = -1
-
                 data = dict(
-                    rpm_val=rpm_average,
+                    rpm_val=list(self._last_rpm_values),
```

Comment:
We are pushing a list to analytics? is that ok in order to query it afterwards or how are we going to check this data afterwards? @Josef-MrBeam